File: JavaScript/Thinkful_-_Logic_Drills_Traffic_light-8_kyu.js
// https://www.codewars.com/kata/58649884a1659ed6cb000072
// 2023-03-09T14:31:18.305+0000
function updateLight(current) {
  const lights = ["green", "yellow", "red"];
  return lights[(lights.indexOf(current) + 1) % lights.length]
}

File: Python/Thinkful_-_Logic_Drills_Traffic_light-8_kyu.py
# https://www.codewars.com/kata/58649884a1659ed6cb000072
# 2023-06-20T16:20:15.962+0000
def update_light(current):
    return ["green", "yellow", "red"][(["green", "yellow", "red"].index(current) + 1) % 3]